libs/hdf-converters/src/utils/attestations.ts

```diff
@@ -2,19 +2,23 @@ import * as XLSX from '@e965/xlsx';
 import {ExecJSON} from 'inspecjs';
 import {
   AttestationData,
-  ControlResultStatus
+  ControlResultStatus,
+  ControlAttestationStatus
 } from 'inspecjs/src/generated_parsers/v_1_0/exec-json';
 import * as _ from 'lodash';
 import moment from 'moment';
-import ms from 'ms';
-
-export type Attestation = {
-  control_id: string;
-  explanation: string;
-  frequency: string;
-  status: 'passed' | 'failed';
-  updated: string;
-  updated_by: string;
+
+// export type Attestation = {
+//   control_id: string;
+//   explanation: string;
+//   frequency: string;
+//   status: 'passed' | 'failed';
+//   updated: string;
+//   updated_by: string;
+// };
+// Convert from using enum type to enum values
```

Comment:
I don't understand this change. Can you explain?

Reply:
The AttestationData interface is what this file should've been using, but the original creator wanted to use strings instead of ControlAttestationStatus enums, so they made their own class for it with a slightly different name. This changes it so it actually uses that original AttestationData while still replacing the status with a string as defined by the ControlAttestationStatus enum:

export enum ControlAttestationStatus {
    Failed = "failed",
    Passed = "passed",
}

I may be misunderstanding this feature though. Credit to @Amndeep7 for the suggestion and change honestly